es6.js

```diff
@@ -0,0 +1,89 @@
+// API:
+// new IntBuilder(int) // constructor takes starting integer, if not passed starts with 0;
+// intBuilder.plus(...n)         // take infinite number of integers and sum all with stored value;
+// intBuilder.minus(...n)        // take infinite number of integers and subtract from stored value; 
+// intBuilder.multiply(n)        // multiply param n on stored value;
+// intBuilder.divide(n)          // leaves integer part of division stored value on n; 
+// intBuilder.mod(n)             // leaves remainder of the division stored value with on n;
+// intBuilder.get()              // returns stored value;
+// random(from, to)   // static method; from, to: integer; values limits the range of random values; 
+
+// EXAMPLE:
+// IntBuilder.random(10, 100);          // 42;
+// const intBuilder = new IntBuilder(10); // 10;
+// intBuilder
+//   intBuilder.plus(2, 3, 2)                     // 17;
+//   intBuilder.minus(1, 2)                       // 14;
+//   intBuilder.multiply(2)                       // 28;
+//   intBuilder.divide(4)                         // 7;
+//   intBuilder.mod(3)                            // 1;
+//   intBuilder.get();                            // -> 1;
+
+class BuilderES6 {
+    constructor(integer) {
+        this.integer = integer || 0;
+    }
+}
+
+class IntBuilder extends BuilderES6 {
```

Comment:
According to the task almost all methods should be chainable. It means that you can call you methods in the following way:
let intBuilder = new IntBuilder(10);
intBuilder.plus(2, 3, 2).minus(1, 2).multiply(2).divide(4).mod(3).get(); 
Right now noone method is chainable. Try to fix that.